modules/network-observability-nodes-taints-tolerations.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// network_observability/network-observability-scheduling-resources.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="network-observability-multi-tenancy{context}"]
+= Using tolerations to control Network Observability deployment in specific nodes
+You can configure the `FlowCollector` to allow you to deploy Network Observability components in specific nodes.
+
+
+`ebpf`:: 
+* `spec.agent.ebpf.advanced.scheduling.NodeSelector`
+* `spec.agent.ebpf.advanced.scheduling.Tolerations`
+* `spec.agent.ebpf.advanced.scheduling.Affinity`
+* `spec.agent.ebpf.advanced.scheduling.PriorityClassName`
+
+`processor`::
+* `spec.processor.advanced.scheduling.NodeSelector`
+* `spec.processor.advanced.scheduling.Tolerations`
+* `spec.processor.advanced.scheduling.Affinity`
+* `spec.processor.advanced.scheduling.PriorityClassName`
+
+`consolePlugin`::
+* `spec.consolePlugin.advanced.scheduling.NodeSelector`
+* `spec.consolePlugin.advanced.scheduling.Tolerations`
+* `spec.consolePlugin.advanced.scheduling.Affinity`
+* `spec.consolePlugin.advanced.scheduling.PriorityClassName`
+
+
+.Sample `FlowCollector` resource for `spec.<component>.advanced.scheduling` 
+[source,yaml]
+====
+apiVersion: flows.netobserv.io/v1beta2
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+# ...
+advanced:
+   scheduling:
+    tolerations:
+    - key: "<taint key>"
+      operator: "Equal"
+      value: "<taint value>"
+      effect: "<taint effect>"
+      nodeSelector:
+        <key>: <value>
+      affinity:
+        nodeAffinity:
+         requiredDuringSchedulingIgnoredDuringExecution:
+           nodeSelectorTerms:
+           - matchExpressions:
+             - key: name
+               operator: In
+               values:
+               - app-worker-node
+      priorityClassName: high-priority
```

Comment:
do u want to include definition of high-priority  or u can use "<priority class name>" ?

Reply:
I think I will just use ""